letsdrill.rb

```diff
@@ -1,13 +1,24 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+ grade = ""
+  grade_ranges = {
+  	"F" => 60,
+  	"D" => 70,
+  	"C" => 80,
+  	"B" => 90
+  }
 
+  grade_ranges.each do |letter, upper_bound|
+  	if (integer < upper_bound)
+  		grade = letter
+  		return grade
+  	else
+  		grade = "A"
+  	end
+  end
 end
 
 def shortest_string(array)
-
-  #Put your code here!
-
+	shortest = array.min { |a, b| a.length <=> b.length }
```

Comment:
awesome.  `shortest` isn't doing anything here; you can omit it.  And check out #min_by.